bears/general/QuotesBear.py

```diff
@@ -0,0 +1,82 @@
+from coalib.bears.LocalBear import LocalBear
+from bears.general.AnnotationBear import AnnotationBear
+from coalib.results.Diff import Diff
+from coalib.results.HiddenResult import HiddenResult
+from coalib.results.Result import Result
+
+
+class QuotesBear(LocalBear):
+
+    BEAR_DEPS = {AnnotationBear}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting'}
+
+    def correct_single_line_str(self, filename, file, sourcerange,
+                                preferred_quotation):
+        """
+        Corrects a given single line string assuming it does not use the
+        preferred quotation. If the preferred quotation mark is used inside the
+        string, no correction will be made.
+
+        This function will yield one or no Result objects.
+
+        :param filename:
+            The filename of the file to correct the line in.
+        :param file:
+            The file contents as list of lines.
+        :param sourcerange:
+            The sourcerange indicating where to find the string.
+        :param preferred_quotation:
+            ``'`` or ``"`` respectively.
+        """
+        str_contents = file[sourcerange.start.line - 1][
+                       sourcerange.start.column:sourcerange.end.column-1]
+
+        if preferred_quotation in str_contents:
```

Comment:
if I prefer double quotes and I have a string `' hey " there '` the method will be entered